salt/orch/kubernetes.sls

```diff
@@ -19,12 +19,20 @@ update_pillar:
     - require:
       - salt: hostname_setup
 
+update_grains:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_grains
```

Comment:
Do we already have a Salt version that supports `refresh_grains`? If that is the case, could you also update `orch/update-etc-hosts.sls` and add it?